test/functional/tests/initialize/test_initialize_load.py

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import time
+
+import pytest
+
+from api.cas import casadm, casctl, casadm_parser, init_config, cas_module
+from api.cas.cache_config import CacheMode
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from test_tools import fs_utils
+from test_tools.disk_utils import Filesystem
+from test_utils.filesystem.file import File
+from test_utils.size import Size, Unit
+
+iterations_per_config = 50
+cas_conf_path = "/etc/opencas/opencas.conf"
+mount_point = "/mnt/cas"
+test_file_path = f"{mount_point}/test_file"
+
+
+@pytest.mark.parametrize("cache_mode", CacheMode)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_stress_load(cache_mode):
+    """
+        title: Initialize test for loading CAS from conf.
+        description: |
+          Start OpenCAS service and check if caches defined in conf file are started properly.
+        pass_criteria:
+          - Cache starts in correct mode.
+    """
+    with TestRun.step("Prepare cache and core devices."):
+        cache_dev = TestRun.disks['cache']
+        cache_dev.create_partitions([Size(100, Unit.MebiByte)])
+        cache_dev = cache_dev.partitions[0]
+        core_dev = TestRun.disks['core']
+        core_dev.create_partitions([Size(200, Unit.MebiByte)])
+        core_dev = core_dev.partitions[0]
+
+    with TestRun.step("Start cache and add core."):
+        cache = casadm.start_cache(cache_dev, cache_mode, force=True)
+        casadm.add_core(cache, core_dev)
+
+    with TestRun.step("Stop cache service."):
+        casadm.stop_all_caches()
+        if len(casadm_parser.get_caches()) != 0:
+            TestRun.fail("Cache did not stop successfully.")
+
+    with TestRun.step("Create configuration for OpenCAS."):
+        fs_utils.create_test_file('/etc/opencas/opencas.conf',
+f"""version=19.9.0
+
+## Caches configuration section
+[caches]
+## Cache ID	Cache device				Cache mode	Extra fields (optional)
+1   {cache_dev.system_path} {cache_mode.name}
+
+## Core devices configuration
+[cores]
+## Cache ID	Core ID		Core device
+1   1   {core_dev.system_path}
+""")
+
```

Comment:
Please consider creating class for opencas_config. It should provide methods for adding caches, cores to config, and saving config to a file on DUT.